source/rust_verify/src/rust_to_vir_expr.rs

```diff
@@ -1286,6 +1286,9 @@ pub(crate) fn expr_to_vir_innermost<'tcx>(
         ExprKind::AddrOf(BorrowKind::Raw, _, _) => {
             unsupported_err!(expr.span, format!("raw borrows"))
         }
+        ExprKind::OffsetOf(_container, _fields) => {
+            unsupported_err!(expr.span, format!("offset_of!()"))
```

Comment:
I don't think this is implementable at the HIR level. It needs to know the layout of a monomorphized type.

Reply:
Thanks for the note. I don't think it's a priority for Verus.